doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,136 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
+```python
+def get_credentials(**kwargs):
+    import os
+    from azure.common.credentials import ServicePrincipalCredentials
+
+    return ServicePrincipalCredentials(
+        client_id = os.environ['AZURE_CLIENT_ID'],
+        secret = os.environ['AZURE_CLIENT_SECRET'],
+        tenant = os.environ['AZURE_TENANT_ID']
+    )
+```
+* Change the `get_azure_core_credentials()` function to construct and return a `ClientSecretCredential:
+```python
+def get_azure_core_credentials(**kwargs):
+    from azure.identity import ClientSecretCredential
+    return ClientSecretCredential(
+        client_id = os.environ['AZURE_CLIENT_ID'],
+        client_secret = os.environ['AZURE_CLIENT_SECRET'],
+        tenant_id = os.environ['AZURE_TENANT_ID']
+    )
+```
 
 **Important: `mgmt_settings_real.py` should not be committed since it contains your actual credentials! To prevent this, it is included in `.gitignore`.**
 
 ## Running tests in live mode
 
 To configure the tests to run in live mode, you have two options:
 
-*   Set the environment variable `AZURE_TEST_RUN_LIVE` to "true" or "yes".
-    If you want to go back to playback mode you can either unset it entirely
-    or set it to "false" or "no".
-*   Create a `testsettings_local.cfg` file in the same directory as
-    `mgmt_settings_real.py`. It should look like the following:
+* Set the environment variable `AZURE_TEST_RUN_LIVE` to "true" or "yes". If you want to go back to playback mode you can either unset it entirely or set it to "false" or "no".
+* Create a `testsettings_local.cfg` file in the same directory as `mgmt_settings_real.py`. It should look like the following:
     ```
     live-mode: true
     ```
-    To go back to playback mode using the config file,
-    change the "true" to "false" or delete the file.
-    (`testsettings_local.cfg` is listed in `.gitignore`
-    and not present in the repo; if it's missing,
-    the tests default to playback mode.)
-
-Now you can run tests using the same method described in
-[Running the tests](#running-the-tests).
-You would be well-advised to specify a limited number of tests to run.
-Running every existing test in live mode will take a very long time
-and produce a great deal of changes to recording files in your Git repository.
+    To go back to playback mode using the config file, change the "true" to "false" or delete the file. (`testsettings_local.cfg` is listed in `.gitignore` and not present in the repo; if it's missing, the tests default to playback mode.)
+
+Now you can run tests using the same method described in [Running the tests](#running-the-tests). You would be well-advised to specify a limited number of tests to run. Running every existing test in live mode will take a very long time and produce a great deal of changes to recording files in your Git repository. However, for changes in the client code, the recordings will need to be committed to the Git repository.
```

Comment:
this might be out of the scope of your test documentation, but do we talk about how to scrub secrets from recordings anywhere?

Reply:
I do talk about scrubbing in the dataplane document, I tried to make as few edits as possible to the mgmt test document.